std/container/rbtree.d

```diff
@@ -801,9 +800,7 @@ if (is(typeof(binaryFun!less(T.init, T.init))))
 
     static private Node allocate(Elem v)
     {
-        auto result = allocate();
```

Comment:
I thought as much. Yeah, `allocate` immediately ringed a bell about plugging std.experimental.allocators in there to me too ;)

No problem, I'm glad that I can now effectively help :)